### PR TITLE
[FEATURE] Migrate Project Statuses to Database Model

### PR DESCRIPTION
**Pull request by Jbithell**
Migrate project statuses to being stored in the database. This PR does not include any functionality to edit those statuses as yet, this will come in a seperate PR

### CODE REVIEW

File: html/admin/api/projects/list.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 $DBLIB->orderBy("projects.projects_dates_deliver_start", "ASC");
 $DBLIB->orderBy("projects.projects_name", "ASC");
 $DBLIB->orderBy("projects.projects_created", "ASC");
-$projects = $DBLIB->get("projects", null, ["projects_id", "projects_archived", "projects_name", "clients_name", "projects_dates_deliver_start", "projects_dates_deliver_end","projects_dates_use_start", "projects_dates_use_end", "projects_status", "projects_manager"]);
```
[the remark this answers is not in the document]
Comment (cherry-john) [reply html/admin/api/projects/list.php:14]: They're definitely needed in the app!

**Review by cherry-john — CHANGES_REQUESTED** [review]
One Api endpoint I think still needs some information that you've removed

File: html/admin/api/projects/list.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jbithell) [reply html/admin/api/projects/list.php:14]: From what I can see these aren't returned by this file, so we need to look post-merge at where this data is coming from in the app

**Review by cherry-john — CHANGES_REQUESTED** [review]
The calendar showing Sub-projects on the projects page doesn't pull through new statuses.

**Pull request merged**